examples/basic/src/model-provider/openai/openai-chat-generate-text-caching-example.ts

```diff
@@ -0,0 +1,26 @@
+import dotenv from "dotenv";
+import { MemoryCache, generateText, openai } from "modelfusion";
+
+dotenv.config();
+
+async function main() {
+  const model = openai.ChatTextGenerator({
+    model: "gpt-3.5-turbo",
+    temperature: 1,
+    maxGenerationTokens: 200,
+  });
+
+  const messages = openai.ChatMessage.system("Write a short story about a robot learning to love:");
+
+  const cache = new MemoryCache();
+
+  const text1 = await generateText(model, [ messages ], { cache } );
+
+  console.log({text1});
```

Comment:
can you run prettier? It should run it automatically on commit if you have set up commit hooks

Reply:
ran prettier. running it did also update the formatting for a couple files I hadn't touched earlier.

Reply:
Awesome. Ideally it would run through `lint-staged.config.js` only on the changed files, but there could be bugs in the config.